src/pj_param.c

```diff
@@ -134,14 +134,15 @@ PROJVALUE pj_param (projCtx ctx, paralist *pl, const char *opt) {
         switch (type) {
         case 'b': case 'i':
             value.i = 0;
-            return value;
+            break;
```

Comment:
This has been changed back and forth a few times now. I don't care much which is used, but a comment here would probably be a good idea to avoid this getting changed back again at a later stage.

Reply:
OK I'll add a comment